warehouse/malware/tasks.py

```diff
@@ -24,3 +27,60 @@ def run_check(task, request, check_name, obj_id):
     except Exception as exc:
         request.log.error("Error executing check %s: %s", check_name, str(exc))
         raise task.retry(exc=exc)
+
+
+@task(bind=True, ignore_result=True, acks_late=True)
+def sync_checks(task, request):
+    code_checks = inspect.getmembers(checks, inspect.isclass)
```

Comment:
I think this could be more specific by calling `MalwareCheckBase.__subclasses__()` rather than relying on `warehouse/malware/checks/__init__.py` to import.

Either that or verifying that the the classes found in the `checks` module are subclasses of `MalwareBaseCheck` directly.

Reply:
I like the `__subclassess__()` suggestion.

Reply:
There is some difficulty with the tests if we add this constraint to the task. Namely, once something is created and added as a subclass, it will persist for the entire execution of the program. This breaks the hermetic isolation of tests. I have preserved the original logic, but added something to `check_tests` to verify that every class exported from `checks` is, in fact, a subclass of `MalwareCheckBase`.

Related:
https://mail.python.org/pipermail/python-list/2003-August/210297.html